### PR TITLE
[feature] Added timer contextmanager for easily timing events

### PR DESCRIPTION
**Pull request by aviramha**

### CODE REVIEW

File: aiodogstatsd/client.py [hunk 1]
```diff
@@ -219,6 +221,22 @@ def _report(
             )
         )
 
+    @contextmanager
+    def timer(
```
Comment (Gr1N) [inline aiodogstatsd/client.py]: 1. Let's name it `timeit`
1. `sample_rate` must be `Optional[typedefs.MSampleRate]` as you already did it in the previous PR
1. And better to rewrite a bit in a way:
```
loop = get_event_loop()
started_at = loop.time()
yield
value = (loop.time() - started_at) * 1000
```

Comment (aviramha) [reply aiodogstatsd/client.py]: What is the benefit of using loop.time?

Comment (Gr1N) [reply aiodogstatsd/client.py]: It's a recommended way to do such things, you can find the same implementation in many libraries or frameworks (e.g. `aiohttp`).

Comment (aviramha) [reply aiodogstatsd/client.py]: I assumed so, but wondered why exactly (to support different timings of event loops?..), oh well :)

**Comment by Gr1N** [comment]
LGTM, thanks!

**Pull request merged**